vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
```

Comment:
Overall this looks great but I wonder if it's worth improving some of the English. There are some passive/indirect structures here (from the original, I think) that feel awkward to me. For example, "These are the minimum infrastructure requirements to install Entando directly on a Kubernetes (K8s) cluster." would be more direct.

By the way, I didn't realize K8s was the correct abbreviation. I know we've used kubernetes and k8s in many places so we should correct that over time...

Reply:
will do